libs/nest/core/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@nrwl/react/babel"],
```

Comment:
Looks like you have used some other Nx plugin generator to generate this library than Nest.
Nest library doesn't require babel and in `.eslintrc.json`, `jest.config.js`, `project.json` and `tsconfig.*` don't need to list `.tsx` or `.jsx` files.

But instead of creating a new library we should just rename the `nest/validators` library and add the `validators` folder.
`yarn nx g @nrwl/workspace:move --project nest-validators --destination nest/core`

Reply:
ah, i just used the previous setup but change the paths to reflect but this seems better